packages/react-button/src/common/isConformant.ts

```diff
@@ -6,11 +6,9 @@ export function isConformant<TProps = {}>(
   testInfo: Omit<IsConformantOptions<TProps>, 'componentPath'> & { componentPath?: string },
 ) {
   const defaultOptions: Partial<IsConformantOptions<TProps>> = {
-    asPropHandlesRef: true,
     componentPath: module!.parent!.filename.replace('.test', ''),
-    disabledTests: ['has-docblock', 'kebab-aria-attributes'],
+    disabledTests: ['kebab-aria-attributes'],
```

Comment:
```suggestion
```

I just tested out, we can also enable `kebab-aria-attributes`:
![image](https://user-images.githubusercontent.com/14183168/156730948-ecf29b19-3cca-44ad-979e-b9c5c53d8944.png)